framework/validators/IpValidator.php

```diff
@@ -0,0 +1,555 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\validators;
+
+use Yii;
+use yii\base\InvalidConfigException;
+use yii\helpers\Json;
+use yii\helpers\StringHelper;
+use yii\web\JsExpression;
+
+/**
+ * IpValidator validates that the attribute value is a valid IPv4/IPv6 address or subnet.
+ * May change attribute's value if normalization is enabled.
+ *
+ * @property boolean|string negationChar whether address may have an negation-character at the beginning
+ *   string - passed character will be used. May be a regular expression
+ *   true   - character from [[DEFAULT_NEGATION_CHAR]] (default `!`) will be used
+ *   false  - negation is not allowed
+ * @author SilverFire <d.naumenko.a@gmail.com>
+ * @since 2.0.5
+ */
+class IpValidator extends Validator
+{
+    /**
+     * @const integer the length of IPv6 address in bits
+     */
+    const IPV6_ADDRESS_LENGTH = 128;
+
+    /**
+     * @const integer the length of IPv4 address in bits
+     */
+    const IPV4_ADDRESS_LENGTH = 32;
+
+    /**
+     * @const integer is used to change check order by [[isAllowed]]
```

Comment:
There is a PHPDOC ```@see order```, you can find detailed description there.
How can I change this doc to make it more clear?